tools/add_module.py

```diff
@@ -213,4 +214,7 @@ def main(argv=None):
 
 
 if __name__ == "__main__":
+  # Under 'bazel run' we want to run within the source folder instead of the execroot.
```

Comment:
I think you'll have to add this to calc_integrity.py and bcr_validation.py, otherwise, they won't work with `bazel run //tools:<name>`

Reply:
Done